version.c

```diff
@@ -393,9 +380,9 @@ void print_version(void)
 #endif
 
 #ifdef HAVE_IDN2_H
-  printf("\nlibidn: %s (compiled with %s)", idn2_check_version(NULL), IDN2_VERSION);
+  printf("%s\n",mutt_idn2_version());
 #elif defined(HAVE_LIBIDN)
-  printf("\nlibidn: %s (compiled with %s)", stringprep_check_version(NULL), STRINGPREP_VERSION);
+  printf("%s\n", mutt_idn_version());
 #endif
```

Comment:
It would be nice to completely hide away the details of IDN.
Here the version code needs to know that there's two possible IDNs.
If we changed the IDN code, we would need to change the version code too.

If we change this to just: `mutt_idna_print_version()`, then all the details will be hidden.